### PR TITLE
fix: adds verified badge to username #2107

### LINKED ISSUE #2107: Username component missing Verified badge on how-to page
The username component has Verified badge across the platform but the badge is missing on how-to page. 

Badge shown on how-to list
image

Badge not showing on how-to page
image

Would be great to have the badge following the username component across the platform :)

### PR DESCRIPTION
**Pull request by thisislawatts**
PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description

Adds back the verified badge when viewing a Howto Article.

Before:  

Screenshot 2023-02-22 at 21 07 54

After:  

Screenshot 2023-02-22 at 21 07 43

## Git Issues

Closes #2107

### CODE REVIEW

**Comment by davehakkens** [comment]
FYI @thisislawatts this feature used to be there. Not sure why it stopped working

**Comment by thisislawatts** [comment]
@davehakkens it was introduced as part of [this PR](https://github.com/ONEARMY/community-platform/pull/1954), in late October '22
https://github.com/ONEARMY/community-platform/commit/7b4defbf2b41e4a5e5fbc0b4dcc11c0f7939cf28#diff-b211df3c286f10adf9962765e7c064709c0990fb9d1071da36292cb5198e61bbR205-R206

I've introduced a test here that explicitly checks for the presence of the verified badge to ensure we have automated checks for this behaviour.

File: src/pages/Howto/Content/Howto/Howto.test.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        <MemoryRouter initialEntries={['/howto/article']}>
+          <Route path="/howto/:slug" exact key={1} component={Howto} />
+        </MemoryRouter>
+      </ThemeProvider>
+      ,
+    </Provider>,
+  )
+
+describe('Howto', () => {
+  beforeAll(() => {})
```
Comment (evakill) [inline src/pages/Howto/Content/Howto/Howto.test.tsx:61]: do we need this?

File: src/pages/Howto/Content/Howto/Howto.test.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  it('shows verified badge', () => {
+    // Act
```
Comment (evakill) [inline src/pages/Howto/Content/Howto/Howto.test.tsx:63]: what does this mean?

**Review by evakill — APPROVED** [review]
looks good! 2 small q's

**Comment by evakill** [comment]
oops still getting used to auto merge.. no biggie

**Comment by thisislawatts** [comment]
Thanks for quick review @evakill, apologies for the auto-merge. I really love it for small PRs but I appreciate it takes some getting used to. 

I will raise a separate PR to address those two snags, I think they are worth blocking a PR for :) 

I must have got used to working with linters that error on empty functions. I forgot that we [removed](https://github.com/ONEARMY/community-platform/pull/1395) the `@typescript-eslint/no-empty-function` rule from this project a while ago. Maybe a nice one to reintroduce if the maintainers are up for it ✨

**Pull request merged**